kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/metadata/node/event/handler/PipelineJobConfigurationChangedProcessor.java

```diff
@@ -24,8 +24,7 @@
 /**
  * Pipeline changed job configuration processor.
  */
-// TODO rename to PipelineJobConfigurationChangedProcessor
-public interface PipelineChangedJobConfigurationProcessor extends TypedSPI {
+public interface PipelineJobConfigurationChangedProcessor extends TypedSPI {
```

Comment:
Keep it for now.
1, It's not decided how to rename it.
2, Refactoring could be in another PR.